### ISSUE TITLE
TMPro not working after importing package

### ISSUE THREAD

**Issue by E-Kaese** [googlesamples/google-signin-unity#95]
After importing this Unity package TMPro does not work anymore.

Unity version 2019.1.0f2
Error:
PrecompiledAssemblyException: Multiple precompiled assemblies with the same name Unity.Compat.dll included for the current platform. Only one assembly with the same name is allowed per platform. Assembly path: {0}
UnityEditor.Scripting.ScriptCompilation.EditorBuildRules.CreateTargetAssemblies (System.Collections.Generic.IEnumerable`1[T] customScriptAssemblies, System.Collections.Generic.IEnumerable`1[T] precompiledAssemblies) (at C:/buildslave/unity/build/Editor/Mono/Scripting/ScriptCompilation/EditorBuildRules.cs:221)
UnityEditor.Scripting.ScriptCompilation.EditorCompilation.UpdateCustomTargetAssemblies () (at C:/buildslave/unity/build/Editor/Mono/Scripting/ScriptCompilation/EditorCompilation.cs:672)
UnityEditor.Scripting.ScriptCompilation.EditorCompilation.SetAllCustomScriptAssemblyJsonContents (System.String[] paths, System.String[] contents, System.String[] guids) (at C:/buildslave/unity/build/Editor/Mono/Scripting/ScriptCompilation/EditorCompilation.cs:892)
UnityEditor.Scripting.ScriptCompilation.EditorCompilationInterface.SetAllCustomScriptAssemblyJsonContents (System.String[] allAssemblyJsonPaths, System.String[] allAssemblyJsonContents, System.String[] guids) (at C:/buildslave/unity/build/Editor/Mono/Scripting/ScriptCompilation/EditorCompilationInterface.cs:241)

**Comment by artplayer2** [comment]
Same here.

**Comment by RFShaun** [comment]
Same and removing it breaks unity completely

**Comment by PollerBoom** [comment]
This is because of the .Net4 issue, removing the "parse" folder from google-signin-unity package could resolve the problem

**Comment by ShikharJunglee** [comment]
Yup, unchecking Parse folder while importing GoogleSignIn.unitypackage is the fix.

**Comment by mkjovanov** [comment]
Not importing the `Parse` folder didn't work in my case.
I have even tried with older versions, but the result is the same.
It's worth noting that besides `TMPro`, Unity's `UnityEngine.UI` namespace is also missing.

**Comment by danielsalyi** [comment]
Same here. Although it works, just blind coding

**Comment by blackmoonstudio** [comment]
Same here
